### DISCUSSION TITLE
Change Temporary Preview Size Dependent On Model

### DISCUSSION THREAD

**Discussion by CNSam** [spatie/laravel-medialibrary#3074] [Q&A]
Hi folks,

Im not sure if I've missed something in the docs or miss-read the instructions.

Ultimately what im trying to achieve is a different temporary preview url dependent on the model that it will eventually be assigned to...is this possible?

For example if I have a Banners module I might want the preview image to be 700x42px whereas a profile upload might only need to show a preview of 60x60px

Tia

**Comment by parallels999** [comment]
> is this possible?

Yes, it is
